### DISCUSSION TITLE
Assigning protonation states with PROPKA

### DISCUSSION THREAD

**Discussion by and-tos** [jensengroup/propka#189] [Q&A]
I'm new to PROPKA and from the documentation I found it's not clear to me if there's a way to add explicit proton atoms to a PDB file, or if the use is limited to the calculation of pKa values for titratable groups. I'd really appreciate any help pointing me in the right direction.

**Comment by sobolevnrm** [comment]
PDB2PQR uses PROPKA to add protons (and other missing atoms) to structures.  It may be what you're looking for:  https://github.com/Electrostatics/pdb2pqr
